test/integration/action/action-integration.spec.js

```diff
@@ -345,6 +350,34 @@ describe('Action Integration Tests', function() {
     });
   });
 
+  describe('Autopilot actions', () => {
+    it('should wait for autopilotReady', async () => {
+      await grpc1.initAutopilot();
+      expect(store1.autopilotReady, 'to be true');
+      await grpc2.initAutopilot();
+      expect(store2.autopilotReady, 'to be true');
+    });
+
+    it('should init autopilot', async () => {
+      await autopilot1.init();
+      expect(store1.settings.autopilot, 'to be true');
+      const status = await grpc1.sendAutopilotCommand('status');
+      expect(status.active, 'to be true');
+    });
+
+    it('should be able to toggle autopilot', async () => {
+      await autopilot1.toggle();
+      expect(store1.settings.autopilot, 'to be false');
+      let status = await grpc1.sendAutopilotCommand('status');
+      expect(status.active, 'to be false');
+
+      await autopilot1.toggle();
+      expect(store1.settings.autopilot, 'to be true');
+      status = await grpc1.sendAutopilotCommand('status');
+      expect(status.active, 'to be true');
+    });
+  });
+
```

Comment:
Also refactored the integration tests and wrapped them into their own describe block.